client/a8c-for-agencies/components/a4a-feedback/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { FormLabel } from '@automattic/components';
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { ChangeEvent, useState } from 'react';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormTextarea from 'calypso/components/forms/form-textarea';
+import IconBad from './icons/bad.svg';
+import IconGood from './icons/good.svg';
+import IconNeutral from './icons/neutral.svg';
+import { FeedbackQueryData } from './types';
+
+import './style.scss';
+
+export type Props = {
+	title: string;
+	description: string;
+	questionDetails: string;
+	ctaText: string;
+	onSubmit: ( data: FeedbackQueryData ) => void;
+	onSkip: () => void;
+};
+
+export function A4AFeedback( {
+	title,
+	description,
+	questionDetails,
+	ctaText,
+	onSubmit,
+	onSkip,
+}: Props ) {
+	const translate = useTranslate();
+	const [ experience, setExperience ] = useState< string >( 'good' );
+	const [ comments, setComments ] = useState< string >( '' );
+
+	return (
+		<div className="a4a-feedback__wrapper">
+			<div className="a4a-feedback__content">
+				<h1 className="a4a-feedback__title">{ title }</h1>
+				<div className="a4a-feedback__description">{ description }</div>
+				<div className="a4a-feedback__questions">
+					<div className="a4a-feedback__question-details">{ questionDetails }</div>
+					<div className="a4a-feedback__experience-selector">
+						<div className="a4a-feedback__experience-label">{ translate( 'Overall' ) }</div>
```

Comment:
Nitpicking: This should be `"a4a-feedback__experience-selector-label` if we follow the class name hierarchy.

Reply:
Done: https://github.com/Automattic/wp-calypso/pull/96934/commits/410a348ba47388de8e9aaa1a3db6627c870f9bf3